migrations/versions/fe12aab5be25_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: fe12aab5be25
+Revises: ebc36833232f
+Create Date: 2021-01-14 11:50:17.757044
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fe12aab5be25'
+down_revision = 'ebc36833232f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('connectors', sa.Column('_api_token', sa.String(length=255), nullable=True))
+    op.add_column('connectors', sa.Column('_password', sa.String(length=255), nullable=True))
+    op.drop_column('connectors', 'password')
```

Comment:
Column should have old names because after this migration all users will lost their connectors. Also, there is no need to use `_` as prefix in column names.